src/azure-cli/HISTORY.rst

```diff
@@ -1620,8 +1620,9 @@ Release History
 
 **Redis**
 
+* Add preview param `--ports` to `az redis force-reboot` command
 * Add preview param `--replicas-per-master` to `az redis create` command
-* Update azure-mgmt-redis from 6.0.0 to 7.0.0rc1
+* Update azure-mgmt-redis from 6.0.0 to 7.0.0rc2
```

Comment:
Please do not write the history manually, it will be automatically generated by PR title.
Guide line: [link](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests)